pkg/apis/iam/v1alpha2/types.go

```diff
@@ -124,9 +128,11 @@ type UserState string
 const (
 	// UserActive means the user is available.
 	UserActive UserState = "Active"
+	// UserPending means the user is not available.
+	UserPending UserState = ""
```

Comment:
No value for UserPending?

Reply:
Currently, user status is a subresource, the state of the user is empty by default which means not synchronized by the user controller.

Reply:
That's not a good way to do that. If you want an empty state by default, it's better to use `*UserState` rather than an empty string value.

Reply:
fixed